modules/gltf/test/gltf-writer.spec.js

```diff
@@ -14,6 +17,31 @@ const REQUIRED_EXTENSION_2 = 'UBER_extension_2';
 const USED_EXTENSION_1 = 'UBER_extension_3';
 const USED_EXTENSION_2 = 'UBER_extension_4';
 
+function checkJson(t, gltfBuilder) {
```

Comment:
Nit; I try to put helpers like this one at the bottom, so the test cases come first.

Reply:
Done in https://github.com/visgl/loaders.gl/pull/970